tests/js/autoAdjustTests.js

```diff
@@ -0,0 +1,94 @@
+/*!
+Copyright 2015 CERTH
+
+Licensed under the New BSD license. You may not use this file except in
+compliance with this License.
+
+You may obtain a copy of the License at
+https://github.com/gpii/universal/LICENSE.txt
+*/
+
+(function ($) {
+    "use strict";
+    
+    fluid.registerNamespace("gpii.tests");
+
+    fluid.defaults("gpii.tests.autoAdjustControls", {
+        gradeNames: ["gpii.adjuster.autoAdjust", "autoInit"],
+        model: {
+            autoAdjust: false
+        }
+    });
+    
+    fluid.defaults("gpii.tests.autoAdjust", {
+        gradeNames: ["fluid.test.testEnvironment", "autoInit"],
+        components: {
+            autoAdjuster: {
+                type: "gpii.tests.autoAdjustControls",
+                container: ".gpii-onOffSwitch-container",
+                options: {
```

Comment:
You just take the contents of the options block, which i think is just the strings block, and move it into the grade/component definition you have on line 16. Let me know if you need any more details.